examples/cpp/infer_multiple_networks/main.cc

```diff
@@ -89,18 +89,22 @@ void PrintVersion() {
 }
 
 int main(int argc, char** argv) {
-  string image = "grace_hopper.jpg";
-  string graph = "inception_v3_2016_08_28_frozen.pb";
+  // parameters below need to modified as per model
+  string image = "3.png";
+  int batch_size = 12;
+  // Vector size is same as the batch size, populating with single image
+  vector<string> images(batch_size, image);
+  string graph = "mnist_inference_quantized_trained_12212018.pb";
   string labels = "";
-  int input_width = 299;
```

Comment:
Please restore the previous example as the mnist_inference_quantized_trained_12212018.pb is not publicly available. Just use the BS=1 as the default.